### PR TITLE
Left menu changes

### PR DESCRIPTION
**Pull request by Kalyani790**
left menu redesign changes

### CODE REVIEW

File: app/assets/translations/appConstants/bebbo/GRda.ts [hunk 1]
```diff
@@ -445,6 +445,12 @@ const GRda = {
                 hcrNoti3: "Health-checkup has been scheduled on {{reminderDateTime}}, please enter Health-checkup details",
                 hcHasScheduled: "Health-check is scheduled",
                 vcHasScheduled: "Vaccination is scheduled",
+                relatedVideoArticle: "Related videos",
+                upto6YearsMsg: "This app is designed for children up to 6 years of age",
+                 //donate button
+                 donateButton:"Donate for Bebbo",
+                 contactUs:"Write to Us",
+                 drawerMenuabtTxtNew: "About {{appName}}",
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/GRda.ts:453]: Can we rename variable to aboutBebbo

File: app/assets/translations/appConstants/bebbo/GRda.ts [hunk 2]
```diff
@@ -445,6 +445,12 @@ const GRda = {
                 hcrNoti3: "Health-checkup has been scheduled on {{reminderDateTime}}, please enter Health-checkup details",
                 hcHasScheduled: "Health-check is scheduled",
                 vcHasScheduled: "Vaccination is scheduled",
+                relatedVideoArticle: "Related videos",
+                upto6YearsMsg: "This app is designed for children up to 6 years of age",
+                 //donate button
+                 donateButton:"Donate for Bebbo",
+                 contactUs:"Write to Us",
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/GRda.ts:452]: May be text is given to you but "u" should be lower case: "Write to us"

File: app/assets/translations/appConstants/bebbo/GRda.ts [hunk 3]
```diff
@@ -445,6 +445,12 @@ const GRda = {
                 hcrNoti3: "Health-checkup has been scheduled on {{reminderDateTime}}, please enter Health-checkup details",
                 hcHasScheduled: "Health-check is scheduled",
                 vcHasScheduled: "Vaccination is scheduled",
+                relatedVideoArticle: "Related videos",
+                upto6YearsMsg: "This app is designed for children up to 6 years of age",
+                 //donate button
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/GRda.ts:450]: Comment is unnecessary

File: app/assets/translations/appConstants/bebbo/ALsq.ts [hunk 4]
```diff
@@ -448,6 +448,10 @@ const ALsq = {
               //24 May
               relatedVideoArticle: "Video të ngjashme",
               upto6YearsMsg: "Ky aplikacion është krijuar për fëmijët deri në moshën 6 vjeç.",
+              //donate button
+              donateButton: "Bëni donacion për Bebbo",
+              contactUs: "Na shkruani",
+              drawerMenuabtTxtNew: "Rheth {{appName}}",
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/ALsq.ts:454]: Text is formatted well here. Can we apply it in below files as well?

File: app/assets/translations/appConstants/bebbo/GRda.ts (hunk 1 above)
Comment (Kalyani790) [reply app/assets/translations/appConstants/bebbo/GRda.ts:453]: ok

File: app/screens/CustomDrawerContent.tsx [hunk 5]
```diff
@@ -54,41 +55,45 @@ import HTML from 'react-native-render-html';
...
     marginRight: 2,
-    padding: 8 
+    padding: 8
   }
 })
-const CustomDrawerContent = ({ navigation }: any):any => {
+const CustomDrawerContent = ({ navigation }: any): any => {
   const { t } = useTranslation();
   const [accordvalue, onChangeaccordvalue] = React.useState(false);
+  const [aboutaccordvalue, onChangeAboutAccordvalue] = React.useState(false);
```
Comment (Akhror) [inline app/screens/CustomDrawerContent.tsx:96]: Should we follow camel case naming convention? 
aboutaccordvalue  > aboutAccordValue
onChangeAboutAccordvalue > onChangeAboutAccordValue

Comment (Kalyani790) [reply app/screens/CustomDrawerContent.tsx]: done

**Pull request merged**